db/migrate/20220818101111_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.references :users, null: false, foreign_key: true
```

Comment:
hello there 👋🏾 , although you have done a fantastic job creating the table and populating it. there is still one tiny 👌🏾  detail left to address. 

you failed to reference the users table with its foreign key constraint `foreign_key : {to_table: another table}` .